src/ucs/config/global_opts.c

```diff
@@ -41,7 +42,9 @@ ucs_global_opts_t ucs_global_opts = {
     .stats_format          = UCS_STATS_FULL,
     .rcache_check_pfn      = 0,
     .module_dir            = UCX_MODULE_DIR, /* defined in Makefile.am */
-    .module_log_level      = UCS_LOG_LEVEL_TRACE
+    .module_log_level      = UCS_LOG_LEVEL_TRACE,
+    .memcpy_min_thresh     = UCS_MEMUNITS_INF,
```

Comment:
1. rename to bultin_memcpy_min, builtin_memcpy_max
2. maybe do it only for x86 for now? on other arch there may be only min thresh, or no thresh at all?
3. default should be UCS_MEMUNITS_AUTO

Reply:
renamed, will be processed x86 only, MEMUNITS_AUTO is set by default